subscription-manager.spec

```diff
@@ -296,6 +296,7 @@ rm -rf %{buildroot}
 # yum plugin config
 %config(noreplace) %attr(644,root,root) %{_sysconfdir}/yum/pluginconf.d/subscription-manager.conf
 %config(noreplace) %attr(644,root,root) %{_sysconfdir}/yum/pluginconf.d/product-id.conf
+%config(noreplace) %attr(644,root,root) %{_sysconfdir}/yum/pluginconf.d/search-disabled-repos.conf
```

Comment:
Should be okay, assuming installing it on < RHEL 7.2 will just quietly skip the plugin if the requested API version isn't available.

If it shows warning or error on stdout/stderr, I'd prefer to make the plugin only installable with the correct yum version. (Or potentially, disabled by default in those cases).